LinaSaysHello/Program.cs

```diff
@@ -13,15 +13,72 @@ internal class Program
     {
         static void Main(string[] args)
         {
-            //Console.WriteLine("Hi! What’s your name?");
-            //string name = Console.ReadLine();
-            //Console.WriteLine("Nice to meet you " + name);
-            //Console.WriteLine("Anastasiya");
-            //Console.WriteLine("Vention");
-            //Console.ReadLine();
+            //Task 1//
+            Console.Write("Enter first number: ");
+            int enteredNr1 = Convert.ToInt32(Console.ReadLine());
 
-           
+            Console.Write("Enter second number: ");
+            int enteredNr2 = Convert.ToInt32(Console.ReadLine());
 
+            Console.Write("Enter one aritmetic operation: +, -, *, /");
+            char aritmeticOperation = Convert.ToChar(Console.ReadLine());
+
+            switch (aritmeticOperation)
+            {
+                case '+':
+                    {
+                        Console.WriteLine("{ 0} + { 1} = {2}", enteredNr1, enteredNr2, aritmeticOperation);
```

Comment:
Task_1 fails with Format Exception (screen 1). 
This happens because the string cannot be properly constructed when variables are passed into it. Because there is an blank space in brackets (screen 2). You have to ways to resolve this issue: remove blank spaces or use string interpolation ))

![image](https://github.com/Linases/LinaSaysHello/assets/55654947/8319bca6-abe2-447b-bf64-2800171ac916)

![image](https://github.com/Linases/LinaSaysHello/assets/55654947/013739fc-a66e-43a1-b0a2-d1c5e98c8643)

Reply:
Also you need to perform mathematical operations on the entered numbers and output result. So third variable in string mustn't be sign of arithmetical operation but the result of entered arithmetical operation (e.g enteredNr1 + enteredNr2)